mne/coreg.py

```diff
@@ -62,6 +63,19 @@
                          os.path.join(surf_dirname, 'lh.smseghead'))
 
 
+def _map_fid_name_to_idx(name: str) -> int:
```

Comment:
this could be a dict, but I was thinking we could easily add some more intelligent checking here in the future, hence i chose a function

Reply:
For fully private functions that we only use internally like this I wouldn't bother. It's fewer lines and just as clear to use a `dict` I think. But we can merge anyway!